atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/ServiceExposureEteTest.java

```diff
@@ -15,10 +15,10 @@
  */
 package com.palantir.atlasdb.ete;
 
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.core.IsNot.not;
-import static org.hamcrest.core.IsNull.nullValue;
+import org.hamcrest.CoreMatchers;
```

Comment:
This is the wrong way to fix this test. Instead, we should import the proper static members:
`Assert.assertThat`
`Matchers.is`
`Matchers.not`
`Matchers.nullValue`.